test/lib/ansible_test/_data/completion/docker.txt

```diff
@@ -1,4 +1,4 @@
-default name=quay.io/ansible/default-test-container:1.9.2 python=3.6,2.6,2.7,3.5,3.7,3.8 seccomp=unconfined
+default name=quay.io/ansible/default-test-container:1.9.3 python=3.6,2.6,2.7,3.5,3.7,3.8 seccomp=unconfined
```

Comment:
Can you split these two updates into separate PRs? We'll merge the default test container change first. We may end up only backporting the default container update.